src/sentry/static/sentry/app/views/organizationEventsV2/transactionView/traceView.tsx

```diff
@@ -16,22 +16,19 @@ type TraceContextType = {
   trace_id: string;
 };
 
-type TransactionViewProps = {
+type PropType = {
   event: Readonly<SentryEvent>;
 };
 
-type TransactionViewState = {
+type StateType = {
   renderMinimap: boolean;
```

Comment:
nit: `showMinimap` instead of `renderMinimap` since we normally use `render-` when it's a custom render function

Reply:
Aside, is this actually needed? Why do we initially set it to false then immediately change it to true on didMount?

Reply:
Good catch. This was used for the old minimap implementation; I've removed this and adjusted the ref placement at https://github.com/getsentry/sentry/pull/14225/commits/d6322d931a25b20f874244d47f0b4e97da239fd1